editor/plugins/asset_library_editor_plugin.cpp

```diff
@@ -962,6 +962,9 @@ HBoxContainer *EditorAssetLibrary::_make_pages(int p_page, int p_page_count, int
 
 	HBoxContainer *hbc = memnew(HBoxContainer);
 
+	if (p_page_count < 2)
```

Comment:
IMO it would be cleaner to check that before calling `_make_pages` to avoid calling the function and instancing an empty HBxContainer needlessly.

Reply:
I'll admit I'm not sure what's the best solution either. @bojidar-bg maybe?

Reply:
Given that they are initialized as empty HBox-es in the constructor as well, the PR-ed approach seems fine :+1:.